smalld/smalld.py

```diff
@@ -30,6 +34,8 @@ def __init__(
         self.base_url = base_url
         self.listeners = []
 
+        self.intents = intents if intents is not None else Intent.all() 
```

Comment:
My only concern here is, when discord stops you from using privileged intents, will this break? vs if it was just left blank it would work, but you'd just not get those events. Maybe just not worth guessing what discord will do.

If we are going to default to all, why default to Intent.all() in the parameter?

Reply:
Wouldn't leaving it blank also break it? becuase the intents will be mandatory in the future. I could as well just remove the default and just make it a required parameter. What do you think?

Reply:
I don't like it as a required parameter. And yeah, I think either of defaulting to all or blank is guessing at what discord will do.

I'm find with default to all, but we can do that in the parameter?

Reply:
We can probably update to a default of all non-privileged intents in the future 🤷

Reply:
OK. I didn't default in the parameter because I've put `Intent` below `SmallD` but I can move it.